src/details/ArborX_DetailsBoundingVolumeHierarchyImpl.hpp

```diff
@@ -341,10 +356,11 @@ void BoundingVolumeHierarchyImpl<DeviceType>::queryDispatch(
     // [ 2 2 2 .... 2 0 ]
     //   ^            ^
     //   0th          Nth element in the view
+    Kokkos::Profiling::pushRegion( "ArborX:BVH:first_pass" );
```

Comment:
Should that one include the memory allocation for the `offset` array?

Reply:
Don't think so.